### PR TITLE
[OptApp] License File update

### PR DESCRIPTION
**Pull request by Igarizza**
**🆕 Changelog**
This pull request includes a change to the `applications/OptimizationApplication/license.txt` file. The most important change is the update to the copyright information.

**📝 Description**
* Updated the copyright information to include new contributors and affiliations.

### CODE REVIEW

**Review by RezaNajian — CHANGES_REQUESTED** [review]
Please add new contributors to a new line after line 4, i mean add to the line 5.

**Comment by Igarizza** [comment]
> Please add new contributors to a new line after line 4, i mean add to the line 5.

done

File: applications/OptimizationApplication/license.txt [hunk 1]
```diff
@@ -1,21 +1,27 @@
 Kratos Optimization Application
 A library based on: Kratos Multi-Physics
 
-Copyright (c) 2022, Reza Najian Asl, Chair of Structural Analysis, Technical University of Munich
+Copyright (c) 2024
+	Suneth Warnakulasuriya *
+	Reza Najian Asl *
+	Ihar Antonau **
+	Roland Wüchner *
```
Comment (philbucher) [inline applications/OptimizationApplication/license.txt:8]: can you reference the commits of this contributor please? :stuck_out_tongue:

**Comment by Igarizza** [comment]
@RezaNajian @sunethwarna If there are no more comments I will merge it=)

**Review by RezaNajian — APPROVED** [review]

**Pull request merged**